wlauto/workloads/multiapp/__init__.py

```diff
@@ -61,15 +61,17 @@ class Multiapp(AndroidUiAutoBenchmark):
     For reliable testing, this workload requires a good and stable internet connection,
     preferably on Wi-Fi.
 
-    NOTE: This workload requires two jpeg files to be placed in the
-    dependencies directory to run.
-
-    WARNING: This workload timings are dependent on the time it takes to sync the Gmail.
-
     Although this workload attempts to be network independent it requires a
     network connection (ideally, wifi) to run. This is because the welcome
     screen UI is dependent on an existing connection.
 
+    SETUP: This workload requires the Google Photos apk to be placed in the
+    dependencies directory and the Gmail and Skype apks to be placed in the
+    multiapp workload directory.
```

Comment:
I don't follow this, sorry.  I get that I should put the photos APK in .workload-automation/multiapp/dependancies/ but where do I put Gmail and Skype?